cmake/external_cldnn.cmake

```diff
@@ -66,5 +66,9 @@ else()
     ExternalProject_Get_Property(ext_cldnn SOURCE_DIR BINARY_DIR)
     add_dependencies(libcldnn ext_cldnn)
     target_include_directories(libcldnn SYSTEM INTERFACE ${SOURCE_DIR}/api)
+
+    if(NOT CMAKE_BUILD_TYPE)
+        set(CMAKE_BUILD_TYPE Release)
+    endif()
```

Comment:
CMAKE_BUILD_TYPE only influences Ninja and Unix Makefiles.
Since nGraph does not support Ninja, you can set default CMAKE_BUILD_TYPE globally to Release if CMAKE_GENERATOR is Unix Makefiles and pass the the value to external projects. 
I think you can do something like

```
diff --git a/CMakeLists.txt b/CMakeLists.txt
index b3c59bdee..2e24dd9a1 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -69,13 +69,16 @@ if ("${CMAKE_GENERATOR}" MATCHES "^Visual Studio.*$")
 endif()

 # Setup CMAKE_ARGS to be forwarded to External Projects
-set(NGRAPH_FORWARD_CMAKE_ARGS
-        -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
-        -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
-        )
-
-if (NOT MSVS)
+set(NGRAPH_FORWARD_CMAKE_ARGS "")
+
+if("${CMAKE_GENERATOR}" STREQUAL "Unix Makefiles")
+     set(NGRAPH_FORWARD_CMAKE_ARGS
+            ${NGRAPH_FORWARD_CMAKE_ARGS}
+            -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+            -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+            )
     if(NOT CMAKE_BUILD_TYPE)
+        set(CMAKE_BUILD_TYPE Release)
         set(NGRAPH_FORWARD_CMAKE_ARGS
                 ${NGRAPH_FORWARD_CMAKE_ARGS}
                 -DCMAKE_BUILD_TYPE=Release
```

Reply:
@silee2 What if it is under Windows? As far as I understand nGraph will support Windows and clDNN supported Windows already.
Can I set this variable outside of the "if (NOT MSVS" clause?

Reply:
@shssf What I'm trying to say is, if you use visual studio for windows, you have to control debug/release in some other way. CMAKE_BUILD_TYPE does not have any impact. msbuild has it's own build directory structure. @rkimballn1 is working on supporting nGraph on windows, so he could help you.